src/Config.php

```diff
@@ -6,7 +6,9 @@
 
 class Config
 {
-    public function __construct(private readonly string $directory)
+    private array $cache = [];
+
+    public function __construct(private readonly string $directory, private readonly bool $useCache = false)
```

Comment:
```suggestion
    public function __construct(
        private readonly string $directory, 
        private readonly bool $useCache = false
    )
```

Reply:
Resolving this as caching actually makes performance worse as per my testing. 

https://github.com/tnapf/Config/pull/1#issuecomment-1567335440